hazelcast/src/test/java/com/hazelcast/nio/serialization/compatibility/ReferenceObjects.java

```diff
@@ -314,4 +321,4 @@ class ReferenceObjects {
             Projections.multiAttribute(aSmallString, aSmallString, anSqlString),
             Projections.identity()
     };
-}
+}
```

Comment:
minor: let's add the newline back

Reply:
Fixed at https://github.com/hazelcast/hazelcast/pull/22716/commits/53d396ba98264c8202a4bb8090d087cf3fcb67fc.